Integrations/SentinelOne-V2/SentinelOne-V2.py

```diff
@@ -1606,7 +1849,16 @@ def threat_to_incident(threat):
         disconnect_agent_from_network()
     elif demisto.command() == 'sentinelone-broadcast-message':
         broadcast_message()
-
+    elif demisto.command() == 'sentinelone-get-events':
+        get_events()
+    elif demisto.command() == 'sentinelone-get-query-id':
```

Comment:
i think the command name here should be `create-query` or something like that
what do you think ?